### PR TITLE
Feature/make susept base rates female not avg

### PR DESCRIPTION
**Pull request by e-lipman**
I meant to make these changes on a different branch, but this is where the updates to the risk adjustment for comorbidities are

### CODE REVIEW

File: src/global_var.R [hunk 1]
```diff
@@ -35,14 +35,23 @@ death_list = c(0.00161, 0.00695, 0.0309,  0.0844,  0.161,   0.595,   1.93,   4.2
...
+#     Notes: lung was below 1 and brought up to 1. Other and immuno remain the pre-June 2020 numbers and are only used if no other condition.
+# death https://www.medrxiv.org/content/10.1101/2020.05.06.20092999v1.full.pdf
+#     Notes: hypertension was below 1 and brought up to 1. Other remains the pre-June 2020 numbers and is only used if no other condition.
+
+
+renal_or    = c(2.6, 5.82, 1.72) # death OR not available in CCDC
+cvd_or      = c(1.4, 4.88, 1.27)
+diabetes_or = c(3.1, 4.57, 1.79)
+hyper_or    = c(1.1, 4.57, 1) # use the same as diabetes
+smoker_or   = c(2.67, 2.64, 1.12) # death OR not available in CCDC
```
Comment (xindyhu) [inline src/global_var.R:51]: Smoker hospitalization OR is 2.3 in the June2020_new_comorbidity_OR.xlsx

File: src/results.R [hunk 2]
```diff
@@ -54,12 +54,25 @@ calculateRisk <- function(input, county_data) {
...
+        icu_odds = icu_odds * eval(parse(text=paste0(condition_root, "_or[2]")))
+      }
+      # Death OR are mutually adjusted except for other - for this one only adjust if it is only condition
+      if ( (!condition_root %in% c("other")) | length(input$conditions)==1 ){
+        death_odds = death_odds * eval(parse(text=paste0(condition_root, "_or[3]")))
+      }
+    }
   }
   if (input$gender == "male") {
     hosp_odds = hosp_odds * male_or[1] # should be hosp_odds(female) * male_or[1]
```
Comment (xindyhu) [inline src/results.R:78]: @e-lipman Do we need to change here to female hospitalization odds? Or is it already taken care of else where?

Comment (e-lipman) [reply src/results.R:78]: @xindyhu good question. hosp_odds, icu_odds, and death_odds are initially set to the female odds in lines 49-51, and then the odds are multiplied up in he same manner as before. So I believe this is correct

Comment (xindyhu) [reply src/results.R:78]: @e-lipman Got it, that's perfect, thanks!

**Review by xindyhu — APPROVED** [review]

**Pull request merged**